libs/application/templates/accident-notification/src/lib/AccidentNotificationTemplate.ts

```diff
@@ -41,6 +42,7 @@ const AccidentNotificationTemplate: ApplicationTemplate<
   name: application.general.name,
   institution: application.general.institutionName,
   readyForProduction: true,
```

Comment:
Guess we can't remove this yet since `undefined` counts as not ready for production, and there are at least 3 templates with `readyForProduction: undefined`...